### ISSUE TITLE
Add Policy Violations Column to Projects Page

### ISSUE THREAD

**Issue by msymons** [DependencyTrack/frontend#94]
### Current Behavior:
The Projects page offers the following columns:

* Project Name
* Version
* Last BOM Import
* BOM Format
* Risk Score
* Active
* Vulnerabilities

ie, no display of policy violations

### Proposed Behavior:
Add a sortable "Policy Violations" column to the page.  This will make it easy to perform tasks such as:
* Sort by risk and check for low policy violaton count...  which may be an indication that policies are incorrectly configured.
* Sort by policy violation count to see the worst offending projects.  When policies are "mature" then these projects should perhaps be focused on first.  

Note that a simple count of total violations would include License and Operability Risk...  so maybe a bit more nuance might be needed.

**Comment by sekwah41** [comment]
While all the data is there to visually add them, to sort them I think would require a backend change to store the policy violations total to store a value like `lastPolicyViolationsTotal`.

I suggest copying the visuals from the vulnerabilities tab and then using a [tooltip](https://bootstrap-vue.org/docs/components/tooltip) to allow users to hover over to get more information about it.

```javascript
{
  policyViolationsAudited: 0,
  policyViolationsFail: 0,
  policyViolationsInfo: 0,
  policyViolationsLicenseAudited: 0,
  policyViolationsLicenseTotal: 0,
  policyViolationsLicenseUnaudited: 0,
  policyViolationsOperationalAudited: 0,
  policyViolationsOperationalTotal: 0,
  policyViolationsOperationalUnaudited: 0,
  policyViolationsSecurityAudited: 0,
  policyViolationsSecurityTotal: 0,
  policyViolationsSecurityUnaudited: 0,
  policyViolationsTotal: 0,
  policyViolationsUnaudited: 0,
  policyViolationsWarn: 0
}
```

Ill make a preview using ![#20a8d8](https://via.placeholder.com/15/20a8d8/000000?text=+), Warn ![#ffc107](https://via.placeholder.com/15/ffc107/000000?text=+) and Fail ![#f86c6b](https://via.placeholder.com/15/f86c6b/000000?text=+)

As having too many different colours may get a bit confusing to be useful information.

I was thinking the violation types may be good originally but then wed need colors to represent violation types which I don't believe there are.

**Comment by sekwah41** [comment]
Proposed view

![image](https://user-images.githubusercontent.com/3209834/150649619-44e5ef94-b530-4324-95fe-4224ab845d6f.png)
Just to also make it a little clearer

**Comment by mykter** [comment]
I think this is complete follow #111?

**Comment by msymons** [comment]
I do not think that it is complete yet because,  although we do not have a "Policy Violations" column on the Projects page, the column is not sortable and thus hard to work with.
